### ISSUE TITLE
JSON output of VK_LAYER_LUNARG_api_dump does not properly escape `"` in string values

### ISSUE THREAD

**Issue by theHamsta** [LunarG/VulkanTools#1791]
I'm currently using version 1.3.239 of the Vulkan SDK. I want to use `VK_LAYER_LUNARG_api_dump` to dump API usage of my Vulkan app. Unfortunately, it does not always print valid JSON when there's a `UNKNOWN` `pNext`. It might be that `"` are not properly escaped via `\"` in string values.

An excerpt from a generated JSON

```
                            "type" : "VkQueueFamilyProperties2",
                            "name" : "[0]",
                            "members" :
                            [
                                {
                                    "type" : "VkStructureType",
                                    "name" : "sType",
                                    "value" : "VK_STRUCTURE_TYPE_QUEUE_FAMILY_PROPERTIES_2"
                                },
                                {
                                    "type" : "const void*",
                                    "name" : "pNext",
                                    "value" : "UNKNOWN ("1000023012")
                                },
```

See the penultimate line, with `"UNKNOWN ("1000023012")` should probably be `"UNKNOWN (\"1000023012\")"`.

I can fix my JSONs by running `sed -i 's/"UNKNOWN.*$/"UNKNOWN"/g' /tmp/apidump.json` (omit everything after UNKNOWN)

**Comment by charles-lunarg** [comment]
This is fixed with PR #1771 
I caught it during the refactor (and about 10 other issues).

**Comment by charles-lunarg** [comment]
Closing as completed.
